src/framework/control.ml

```diff
@@ -18,12 +18,23 @@ let spec_module: (module Spec) Lazy.t = lazy (
   let arg_enabled = get_bool "witness.graphml.enabled" || get_bool "exp.arg.enabled" in
   let termination_enabled = List.mem "termination" (get_string_list "ana.activated") in (* check if loop termination analysis is enabled*)
   let open Batteries in
+  let gas () =
```

Comment:
Maybe this could be moved to some of the files defining the gas?

Reply:
I now succeeded moving almost everything into `constraints.ml`. Is this cleaner now?

Reply:
Actually I even tried having a new file for this now. Maybe this is helpful to not make `constraints.ml` too large?

Reply:
Indeed, not dumping everything into `Constraints` is a good idea.